src/bluecoins_cli/cli.py

```diff
@@ -62,3 +63,19 @@ async def convert(
             click.echo(f"==> account {id_}: {q(rate)} {currency} -> {q(true_rate)} {base_currency}{currency}")
 
     cache.save()
+
+@cli.command(help='Create Archive account')
+@click.pass_context
+async def create_archive(
+    ctx: click.Context
+) -> None:
+    conn = open_copy(ctx.obj['path'])
+
+    cache = QuoteCache(os.path.join(xdg.xdg_cache_home(), 'bluecoins-cli', 'quotes.json'))
+    cache.load()
+
+    create_Archive_account(conn)
```

Comment:
The transaction is not committed. Wrap with `transaction(conn)` as in convert command.